proto/attestation/verification.proto

```diff
@@ -47,3 +49,144 @@ message AttestationResults {
   // Elliptic-Curve-Point-to-Octet-String conversion.
   bytes signing_public_key = 4;
 }
+
+// Evidence values extracted from a DICE certificate chain during verification.
+message ExtractedEvidence {
```

Comment:
I don't think `Readings` is better. `Measurement` might be a bit better, but is not accurate, since not everything are measurements. I could not think of any other better options. Any other suggestions?

Reply:
`Measurement` would be fine as well. Personally, I'd rank consistency higher than accurateness. I suggested `Readings` mostly because it is an unused word which roughly captures the idea.

Other suggestions:

`Quantities`
`Measures`
`Gauges`
`Analysis`
`Assessments`

Reply:
I went with "Data" in the end, since it doesn't conflict with "ReferenceValues" and is more accurate. I personally want message/type names to reflect the contents as accurately as possible.